src/imitation/scripts/common/train.py

```diff
@@ -0,0 +1,220 @@
+"""Common configuration elements for training imitation algorithms."""
+
+import logging
+import os
+from typing import Any, Mapping, Optional, Sequence, Tuple
+
+import sacred
+from stable_baselines3.common import base_class, vec_env
+
+from imitation.data import rollout, types
+from imitation.policies import base
+from imitation.util import logger as imit_logger
+from imitation.util import sacred as sacred_util
+from imitation.util import util
+
+train_ingredient = sacred.Ingredient("train")
+logger = logging.getLogger(__name__)
+
+
+@train_ingredient.config
+def config():
+    # Logging
+    log_root = None
+    log_dir = None
+    log_format_strs = ["tensorboard", "stdout"]
+
+    # Environment config
+    env_name = "seals/CartPole-v0"  # environment to train on
+    num_vec = 8  # number of environments in VecEnv
+    parallel = True  # Use SubprocVecEnv rather than DummyVecEnv
+    max_episode_steps = None  # Set to positive int to limit episode horizons
+    env_make_kwargs = {}  # The kwargs passed to `spec.make`.
+
+    # Demonstrations
+    data_dir = "data/"
+    rollout_path = None  # path to file containing rollouts
+    n_expert_demos = None  # Num demos used. None uses every demo possible
+
+    # Trainig
+    # TODO(adam): does this need to be here or could it be in rl and separately for bc?
+    # Is there any script we want to not take a policy?
+    # Hmm, MCE IRL doesn't need this -- it can just ignore it though.
+    # (Perhaps have that script warn if something non-default is set...?)
+    policy_cls = base.FeedForward32Policy
+    policy_kwargs = {}
+
+    # Evaluation
+    n_episodes_eval = 50  # Num of episodes for final mean ground truth return
+
+    # TODO(adam): should we separate config into a separate file?
+    # or just disable F841 on this whole file?
+    _ = locals()  # quieten flake8
+    del _
```

Comment:
I mean this:

```python
# This is redundant:
_ = locals()
del _

# Instead:
locals()
```

(disclaimer: I only tested this quickly in my editor and am not certain it will work for all versions of flake8)

Reply:
Thanks, this works, will make that change!